.github/workflows/google-java-format.yml

```diff
@@ -0,0 +1,46 @@
+#####################################################################################
+# GitHub Action to format specific files by google-java-format.
+#
+# Workflow starts when:
+# 1) push to master
+# 2) PR created or pushed
+#
+#####################################################################################
+name: Google-Java-Format
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: '*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    if: github.repository == 'checkstyle/checkstyle'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v4
+        with:
+          java-version: 11
+          distribution: 'temurin'
+
+      - name: Checkout Pull Request Code
+        uses: actions/checkout@v4
+
+      - uses: robinraju/release-downloader@v1
+        with:
+          repository: 'google/google-java-format'
+          tag: 'v1.22.0'
```

Comment:
`1.22.0` is used 3 times in this file, does it make sense to move it into env var?

Reply:
done